cmd/format-coverage.go

```diff
@@ -32,7 +33,7 @@ type CoverageFormatter struct {
 var formatOptions = CoverageFormatter{}
 
 // a prioritized list of the formatters to use
-var formatterList = []string{"simplecov", "lcov", "coverage.py", "clover", "gocov", "gcov", "cobertura"}
+var formatterList = []string{"simplecov", "lcov", "coverage.py", "clover", "gocov", "gcov", "cobertura", "jacoco"}
```

Comment:
Don't need to do it in this PR, but could we alphabetize this list and the Map below? Would that hurt?